vitrina/compat/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+
+class Redirections(models.Model):
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
+    name = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
```suggestion
    name = models.CharField(max_length=255, blank=True)
```

Is there are any reasons for `null=True`, why `blank=True` is not sufficient? There is a general rule, to never add nullable strings, because this will complicate handling strings. Nullable strings can be added if there is a good reason for it and empty strings have a meaning.